bin/lp-builder.js

```diff
@@ -22,12 +22,13 @@ var config = {
   LP_VERSION: null,
   LP_APPS: null,
   LP_TASKS: ['copy', 'appmanifests', 'optimize'],
+  LP_GLOBAL_TASKS: ['copySpeechData'],
```

Comment:
OK, that makes sense.  Can we rename `LP_TASKS` to `LP_APP_TASKS` and `LP_GLOBAL_TASKS` to `LP_LOCALE_TASKS`?  I can also to this in a follow-up.